cli/src/org/partiql/shell/Shell.kt

```diff
@@ -79,6 +79,7 @@ class Shell(
     private val parser: Parser,
     private val compiler: CompilerPipeline,
     private val initialGlobal: Bindings<ExprValue>,
+    private val isMonochrome: Boolean = false
```

Comment:
Probably fine for now, I am keen on parsing all options to a configuration object and passing that rather than individual options